src/bindings/jna/plugins/sorted/README.md

```diff
@@ -0,0 +1,34 @@
+> - infos = Sorted Java Plugin, loaded by process plugin
+> - infos/author = Markus Bointner <e11808221@student.tuwien.ac.at>, Philipp Leeb <e11808219@student.tuwien.ac.at>
+> - infos/license = BSD
+> - infos/provides = check
+> - infos/placements = presetstorage postgetstorage
+> - infos/metadata = check/sorted check/sorted/direction
+> - infos/description = Enforces a given order of array elements based on a custom defined key or primitive values
```

Comment:
Description must be last (because the rest of the README.md belongs to the description)